test/acceptance/behave/components/ngsiv2/api_entry_point/retrieve_api_resource.feature

```diff
@@ -45,7 +45,7 @@ Feature: general operations in Context Broker using NGSI API v1 and v2
   Actions After the Feature:
   Setup: stop ContextBroker
 
-  @version
+  @version @skip
```

Comment:
Maybe XML removal has affected this test case, now /version works always in JSON. However, I understand that the right approach would be fixing the test instead of skipping. I'd recomment fixing in this very PR.